hw/xfree86/common/xf86.h

```diff
@@ -90,6 +90,14 @@ extern _X_EXPORT ScrnInfoPtr xf86ConfigPciEntity(ScrnInfoPtr pScrn,
                                                  EntityProc enter,
                                                  EntityProc leave,
                                                  void *private);
+#else
+#define xf86VGAarbiterInit() do {} while (0)
```

Comment:
why is xfree86 built at all on cygwin ?

Reply:
@metux the `dummy` driver makes it possible to have a virtual X11 server not backed by a GPU - similar to Xvfb, but with some advantages.

Reply:
Mostly to do with the fact that the `dummy` driver supports randr 1.6 and Xvfb does not, so you can add / remove / re-configure outputs on the fly, including configuring the size (in mm) of the virtual monitors - for applications that calculate their own DPI.

Pointers on how this used in xpra can be found here:
https://github.com/Xpra-org/xpra/issues/3524
and here:
https://github.com/Xpra-org/xpra/blob/master/docs/Features/DPI.md

FYI: be aware that running a secondary X11 server using the dummy driver is problematic on Debian due to some tty patches they carry that cause other X11 servers on the same system to crash:
https://github.com/Xpra-org/xpra/issues/2834#issuecomment-1179497367